test/example/alpha.md

```diff
@@ -0,0 +1,12 @@
+|    flag |     f(x) |    time |  iter |
+|---------|----------|---------|-------|
+| failure | -6.9e-01 | 6.2e+01 |    70 |
```

Comment:
Something happened here because the values of f(x) are printed using `%8.1e` even though your default format was `%7.1e`. Does `pretty_table()` perform any additional formatting?

Reply:
`pretty_table` adjusts the width and aligns to the left, so it seems like it used `8`.

Reply:
Ok. We should still use `%8.1e` by default so it works with negative numbers.